ert/storage/_storage.py

```diff
@@ -299,11 +300,14 @@ async def add_record_metadata(
 
 
 async def transmit_record_collection(
```

Comment:
We might split this function into two: 
1) with atomic records (no awaitable) 
2) bigger records like tar with await